src/components/login/container/index.jsx

```diff
@@ -30,7 +30,7 @@ export class Login extends Component {
 
     if (loggedIn) {
       notify.show(`Logged in successfully as ${this.state.username}`, 'success', 3000);
-      this.props.history.push('/dashboard');
+      this.props.history.push('/home');
```

Comment:
Please use destructuring props assignment.

Reply:
Thanks for the observation. Will do that.